zone/effects.cpp

```diff
@@ -96,6 +95,10 @@ int64 Mob::GetActSpellDamage(uint16 spell_id, int64 value, Mob* target) {
 			ratio += RuleI(Spells, WizCritRatio); //Default is zero
 		}
 
+		if (IsClient() && (spell_id == SPELL_HARM_TOUCH || spell_id == SPELL_HARM_TOUCH2 || spell_id == SPELL_IMP_HARM_TOUCH )) {
```

Comment:
There's a few places in the code where we are doing this

```
spell_id == SPELL_HARM_TOUCH || spell_id == SPELL_HARM_TOUCH2 || spell_id == SPELL_IMP_HARM_TOUCH
```

We can simplify this to a helper

```cpp
bool Mob::IsHarmTouchSpell(uint16 spell_id)
{
	return spell_id == SPELL_HARM_TOUCH ||
		   spell_id == SPELL_HARM_TOUCH2 ||
		   spell_id == SPELL_IMP_HARM_TOUCH;
}
```

And use it in place